integreat_cms/cms/migrations/0073_add_organization_website.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.19 on 2023-07-04 14:49
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add website to organization
+    """
+
+    dependencies = [
+        ("cms", "0072_alter_poicategory_icon"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="organization",
+            name="website",
+            field=models.URLField(default="", max_length=250, verbose_name="website"),
+        ),
```

Comment:
```suggestion
        ),
        migrations.AlterField(
            model_name="organization",
            name="website",
            field=models.URLField(max_length=250, verbose_name="website"),
        ),
```
Directly after creating the field and populating the existing entries with empty strings, I would alter the field so we don't have an empty string for new objects.